workflow/pred_temperature_ridge.ipynb

```diff
@@ -0,0 +1,436 @@
+{
```

Comment:
<p>Also, we can get a list of all intervals by running the following line. There, you will find the intervals <code>-1</code> and <code>1</code>, which corresponds to the creation of a precursor interval (negative integer) and a target interval (positive integers), respectively.</p><p> </p><p>Change made in commit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>